### PR TITLE
Add esm build

### PR DESCRIPTION
**Pull request by fsher**

### CODE REVIEW

**Review by cbosborn — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -2,16 +2,28 @@
...
-  "types": "./lib/es/index.d.ts",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "types": "./lib/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    },
+    "./index.css": "./lib/index.css"
```
Comment (Kiryous) [inline package.json:17]: there is no need for css here

File: tsconfig.json [hunk 2]
```diff
@@ -1,23 +1,40 @@
 {
+  "exclude": [
+    "node_modules",
+    "lib",
+    "packages/*/*/node_modules",
```
Comment (Kiryous) [inline tsconfig.json:5]: we don't need packages/*/** here. Assuming it would be removed with the general clean-up

**Review by Kiryous — APPROVED** [review]
A few tweaks to cleanup and re-introduce wildcards, otherwise — good to go

File: package.json (hunk 1 above)
Comment (fsher) [reply package.json]: ++ thanks!

File: tsconfig.json (hunk 2 above)
Comment (fsher) [reply tsconfig.json]: Yes sir, thanks! Copy-paste...

**Pull request merged**